queries.sql

```diff
@@ -1,10 +1,83 @@
 /*Queries that provide answers to the questions from all projects.*/
-
 SELECT * FROM animals WHERE name LIKE '%mon';
 SELECT * FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
 SELECT * FROM animals WHERE neutered=true AND escape_attempts<3;
 SELECT date_of_birth FROM animals WHERE name='Agumon' OR name='Pikachu';
 SELECT name,escape_attempts FROM animals WHERE weight_kg>10.5;
 SELECT * FROM animals WHERE neutered=true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
-SELECT * FROM animals WHERE weight_kg<=17.3 AND weight_kg>=10.4;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg<=17.3 AND weight_kg>=10.4;
+
+/* First Transation*/
+BEGIN WORK;
+ALTER TABLE animals 
+RENAME COLUMN species TO unspecified;
+SELECT * FROM animals;
+ROLLBACK;
+END;
```

Comment:
- You have done well so far :+1: :+1: But there are a few changes and requirements you need to meet before moving to the next milestone. :+1: :waffle: 

>Inside a transaction update the animals' table by setting the species column to unspecified. Verify that the change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.

- [ ] It seems you didn't get the instructions clearly for this query. According to the project requirement, you are meant to set the value of the `species` column for all the animals to `unspecified`. You are not meant to rename the COLUMN but set the value of the data in the COLUMN. I suggest you update this query to include the two verifications. here's what it should look like. :+1: :+1: 

```sh
   BEGIN;
   UPDATE animals
   SET species = 'unspecified';
   SELECT species from animals; 
   ROLLBACK;
   SELECT species from animals;
```

Reply:
Upsss! thank  @V-Blaze